src/components/header.tsx

```diff
@@ -1,18 +1,51 @@
 import type { NextPage } from 'next';
+import Link from 'next/link'
+
 
 const Header: NextPage = () => {
-    const style = {
-        backgroundColor: 'blue',
-        color: 'white',
-        width: '100%',
-        height: '50px'
-    }
     return (
-        <div className='header' style={style}>
-            <div style={{ margin: '10px' }}>
-                CS: 498 - SAP
+        <nav className="sticky p-6 mx-auto bg-white w-full">
```

Comment:
Not much to add here. But this looks good. Nice work.

The signup and login buttons could be tested, however. Nothing crazy, but verifying the routing works as expected.

Reply:
As discussed, we won't be worrying about testing for the frontend due to time constraints.